Go/README.md

```diff
@@ -0,0 +1,100 @@
+# Azure Management Go SDK Samples for Azure Cosmos DB
+
+Repository of Azure Management Go SDK samples for creating and updating Azure Cosmos DB resources.
+
+## Azure SDK for Python for Azure Cosmos DB
+
+You can find the source code and for this and other Azure SDKs at:
+[azure-mgmt-cosmosdb](https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/cosmos/azure-mgmt-cosmosdb)
```

Comment:
can you change the package name and URL for the azure sdk for go repo?

Reply:
Updated